selfservice/flow/login/flow_test.go

```diff
@@ -31,7 +32,7 @@ func TestFakeFlow(t *testing.T) {
 	var r login.Flow
 	require.NoError(t, faker.FakeData(&r))
 
-	assert.NotEmpty(t, r.ID)
+	assert.Equal(t, uuid.Nil, r.ID)
```

Comment:
You can also use `assert.NotZero`, because `uuid.Nil` is the zero value of a `uuid.UUID`

Reply:
Ah, nice!